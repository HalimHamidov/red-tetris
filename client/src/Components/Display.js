import React from "react"
import { Styledisplay, StyledTitle, StyledData } from "./styling/StyledDisplay"

const Display = ({ gameover, title, data }) => (
    <Styledisplay gameOver={gameover}>
        <StyledTitle>{title}</StyledTitle>
        <StyledData>{data}</StyledData>
    </Styledisplay>
)

export default Display;