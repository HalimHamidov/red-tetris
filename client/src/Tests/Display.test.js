import renderer from "react-test-renderer";
import Display from "../Components/Display";

test("Test Display rendering", () => {
    const disr = renderer.create( < Display / > ).toJSON();
    expect(disr).toMatchSnapshot();
});

test("Test Display title rendering", () => {
            const disr = renderer.create( < Display title = "tetris"
                gameover = { true }
                />).toJSON();
                expect(disr).toMatchSnapshot();
            });