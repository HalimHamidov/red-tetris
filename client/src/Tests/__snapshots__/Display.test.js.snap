// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Display rendering 1`] = `
<div
  className="sc-bdvvtL epyCWs"
>
  <div
    className="sc-dkPtRN fdlklY"
  />
  <div
    className="sc-hKwDye jjPGBA"
  />
</div>
`;

exports[`Test Display title rendering 1`] = `
<div
  className="sc-bdvvtL gQorsJ"
>
  <div
    className="sc-dkPtRN fdlklY"
  >
    tetris
  </div>
  <div
    className="sc-hKwDye jjPGBA"
  />
</div>
`;
